bio/example_test.go

```diff
@@ -0,0 +1,234 @@
+package bio_test
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/TimothyStiles/poly/bio"
+	"github.com/TimothyStiles/poly/bio/fasta"
+	"github.com/TimothyStiles/poly/bio/genbank"
+	"github.com/TimothyStiles/poly/bio/gff"
+	"github.com/TimothyStiles/poly/bio/polyjson"
+)
+
+// This is where the integration tests that make effed up cyclic dependencies go.
+func Example() {
+	// Poly can take in basic gff, gbk, fasta, and JSON.
+	// We call the json package "pson" (poly JSON) to prevent namespace collision with Go's standard json package.
+
+	gffInput, _ := gff.Read("../data/ecoli-mg1655-short.gff")
+	gbkInput, _ := genbank.Read("../data/puc19.gbk")
+	//fastaInput, _ := fasta.Read("fasta/data/base.fasta")
+	jsonInput, _ := polyjson.Read("../data/cat.json")
+
+	// Poly can also output these file formats. Every file format has a corresponding Write function.
+	_ = gff.Write(gffInput, "test.gff")
+	_ = genbank.Write(gbkInput, "test.gbk")
+	//_ = fasta.WriteFile(fastaInput, "test.fasta")
```

Comment:
why is fasta commented out?

Reply:
No `Write` functions anymore (see: `Example_writeAll()`)

Reply:
We *could* reimplement them, but the type checking will kinda suck, or we'd have to be super verbose. The reason is that `[]*DataType` does not equal `[]interface{}`, even if `*DataType` DOES equal `interface{}`. Because its a slice, you have to either accept `any` as an input, or you have to do an interface conversion beforehand, which is kinda the worst.

Alternatively, they all implement `io.WriterTo`, so you can just write it to a document using a standard-library  interface.